atm.js

```diff
@@ -1,3 +1,34 @@
+var checkingBalance = 0;
+//var custCheckingDeposit = 5 //$("#checking>.input").val()
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
 });
+
+$ ("body").css ("background-color", "red");
+
++//object with methods to check balance and change color if zero/not zero
+var balanceCheck = {
+  checking: function(){
+      if (checkingTotal == 0) {
+        $("#checking > .balance").addClass("zero")
+      } else {
+        $("#checking > .balance").removeClass("zero")
+      }
+    },
+//checkingBalance = checkingBalance + custCheckingDeposit
```

Comment:
I would focus on breaking down the problem, and getting one piece to work at a time, starting with the simple stuff. What you have looks like you were engaging with the lab in a meaningful way, I just see some scattered efforts.